src/Microsoft.DotNet.Interactive.Http.Parsing/Parsing/DynamicExpressionUtilities.cs

```diff
@@ -0,0 +1,289 @@
+﻿using Microsoft.DotNet.Interactive.Http;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+using System.Linq;
+using System.Text;
+using System.Text.RegularExpressions;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.DotNet.Interactive.Http.Parsing
+{
+    #nullable enable   
+    internal class DynamicExpressionUtilites
+    {
+        internal static HttpBindingResult<object?> MatchExpressionValue(HttpExpressionNode node, string expression)
```

Comment:
The name is a little confusing. Can you come up with a name that makes it clearer what this method is actually doing?